### ISSUE TITLE
[NTR] Intermediary layers of neocortex

### ISSUE THREAD

**Issue by shawntanzk** [obophenotype/uberon#2084]
Cell types in BDS have the following layer information that needs to be added into UBERON:
L2/3
L4/5
L5/6
L6b

For the first 3 I suggest the following: 
Label: cortical layer X/Y etc. 
Definition: A layer of the neocortex that is located between layers X & Y
SubclassOf: layer of neocortex

For L6b: 
Label: cortical layer VIb
Definition: The deepest layer of the neocortex. Neurons in L6b projects to cortical targets and higher-order thalamus . [PMID:32160552]
SubclassOf cortical layer VI
Synonyms: neocortex layer 6b, cortical L6b

Related to: https://github.com/obophenotype/cell-ontology/pull/1270

**Comment by dosumis** [comment]
> Definition: A layer of the neocortex that is located between layers X & Y

Are you sure?  I thought is was being used to refer to the sum of layers X and Y

**Comment by shawntanzk** [comment]
yeah I think sum might be better - I understand it as lower X & upper Y which technically isn't in between I guess. How about instead:  A layer of the neocortex that is comprises of both layers X & Y

**Comment by dosumis** [comment]
> Label: cortical layer VIb
> Definition: The deepest layer of the neocortex. Neurons in L6b projects to cortical targets and higher-order ? thalamus . [PMID:32160552]
> SubclassOf cortical layer VI

subclassing probably OK here, but we need a consistent pattern for subClassOf vs part_of 

Need to decide: Is a combination of 2 adjacent layers still a layer? (Could be if the general class is defined right).  Is VIb a subclass of VI or part of it?

**Comment by shawntanzk** [comment]
I think VIb is both part of and subclass of VIb? (not 100% sure what the difference is in this case). From reading, VIb is a distinct part of a VI that has specific morphologically, genetically, and developmentally neurons. Perhaps a distinct feature (that now that I look at it should probs be in def) is that L6b neurons are derived from a precursor population that arises from the subplate.
